src/utils/HypervBoxUtil.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+import path from 'path';
+import util from './Util';
+import Promise from 'bluebird';
+
+var HypervBox = {
+  command: function () {
+    let hypervPath = "C:\Program Files\Hyper-V";
+    if (util.isWindows()) {
+        if (HypervBox.pathExists())
+        {
+            return hypervPath;
+        }
+    }
+     
+    return "";
+  },
+  pathExists: function (){
+    let hypervPath = "C:\Program Files\Hyper-V";
+
+    try {
+        fs.accessSync(hypervPath, fs.F_OK);
+        
+        return true;
+    } catch (e) {
+        return false;
+    }
+  },
+  installed: function () {
+    if (util.isWindows() && !HypervBox.pathExists()) {
+      return false;
+    }
+    return fs.existsSync(this.command());
+  },
+  active: function () {
+    return fs.existsSync('/dev/vboxnetctl');
+  },
+  version: function () {
+    return util.execFile([this.command(), '-v']).then(stdout => {
+      let matchlist = stdout.match(/(\d+\.\d+\.\d+).*/);
```

Comment:
imo won't work: Command to get hyper-v versions is: `Get-VMHostSupportedVersion`

for me it returns: 

```
PS C:\WINDOWS\system32> Get-VMHostSupportedVersion

Name                                                 Version IsDefault
----                                                 ------- ---------
Microsoft Windows 8.1/Server 2012 R2                 5.0     False
Microsoft Windows 10/Server 2016 Technical Preview 3 6.2     False
Microsoft Windows Server 2016 Technical Preview 4    7.0     True
```